metaflow/client/filecache.py

```diff
@@ -231,45 +215,39 @@ def _get_backend_datastore_cls(ds_type):
 
     def _get_flow_datastore(self, ds_type, ds_root, flow_name):
         cache_id = self._flow_ds_type(ds_type, ds_root, flow_name)
-
         cached_flow_datastore = self._store_caches.get(cache_id)
+
         if cached_flow_datastore:
             return cached_flow_datastore
+        else:
```

Comment:
right, it's a stylistic thing to avoid early returns. It is nice when the return value of a function is the value of the last expression evaluated (`if-else` in this case) like in many functional languages.

Returns in the middle of the function body can make it harder to follow the control flow. Obviously not a huge deal.

Reply:
Sure, I don't have a preference. There are both behaviors in the code. Something for black to check maybe :). One of the linters warns you about it (or is it in Go, I dont' remember) -- anywyas, like you say, not a huge deal either way.